crates/bevy_render/src/camera/bundle.rs

```diff
@@ -83,7 +83,7 @@ pub struct OrthographicCameraBundle<M: Component> {
 impl OrthographicCameraBundle<Camera3d> {
     pub fn new_3d() -> Self {
         let orthographic_projection = OrthographicProjection {
-            scaling_mode: ScalingMode::FixedVertical,
+            scaling_mode: ScalingMode::FixedVertical(2.0),
```

Comment:
I don't like that we have to use 2.0 here :( I'm not sure how to proceed though.

Reply:
Another alternative would be to add `scale: 2` instead, but it does not seem any better